airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -19,8 +19,25 @@
 public class PostgresTestDatabase extends
     TestDatabase<PostgreSQLContainer<?>, PostgresTestDatabase, PostgresTestDatabase.PostgresConfigBuilder> {
 
-  static public PostgresTestDatabase in(String imageName, String... methods) {
-    final var container = new PostgresContainerFactory().shared(imageName, methods);
+  public static enum BaseImage {
+
+    POSTGRES_16_BULLSEYE("postgres:16-bullseye"),
+    POSTGRES_12_BULLSEYE("postgres:12-bullseye"),
+    POSTGRES_16_ALPINE("postgres:16-alpine"),
```

Comment:
not a blocking comment but I'd appreciate it if this stack contained a change where POSTGRES_16_ALPINE is removed in favour of POSTGRES_16_BULLSEYE